src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -137,7 +138,8 @@ private TableOfContents(PDDocument document, Bundle bundle) throws IOException {
             }
 
             addCenterText(document, getPage(), INDEX_PAGE, 130);
-            addText(document, getPage(), PAGE, 480,165, PDType1Font.HELVETICA,12);
+            String pageText = bundle.getPageNumberFormat().name().equalsIgnoreCase(PageNumberFormat.pageRange.name()) ? PAGE : TOTAL_PAGES;
```

Comment:
I don't like introducing conditional logic here that is specific to certain PageNumberFormat implementations

Reply:
Perhaps it's better to implement a new method on the enums